src/lib.rs

```diff
@@ -34,16 +34,14 @@ extern crate rusoto_core;
 extern crate rusoto_firehose;
 extern crate seahash;
 extern crate serde;
+#[macro_use]
```

Comment:
@pulltab @tsantero now that we're using the `json!` macro we need to import that macro from `serde_json`. Previously we weren't using it.